### PR TITLE
Deprecated validator -> renamed validator, new deprecated validator

### PR DESCRIPTION
**Pull request by jonsnyder**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have made any necessary test changes and all tests pass.
- [ ] I have run the Sandbox successfully.

### CODE REVIEW

File: src/components/ActivityCollector/configValidators.js [hunk 1]
```diff
@@ -36,9 +36,7 @@ export default objectOf({
...
   }),
   downloadLinkQualifier,
-  onBeforeLinkClickSend: callback()
-}).deprecated(
-  "clickCollection.filterClickedElementProperties",
-  callback(),
-  "onBeforeLinkClickSend"
-);
+  onBeforeLinkClickSend: callback().deprecated(
+    'The field "onBeforeLinkClickSend" has been deprecated. Use "clickCollection.filterClickedElementProperties" instead.'
```
Comment (jfkhoury) [inline src/components/ActivityCollector/configValidators.js:40]: Seems like Martin decided on filterClickDetails.

**Review by jfkhoury — APPROVED** [review]
A quick rename ;)

File: src/components/ActivityCollector/configValidators.js (hunk 1 above)
Comment (liljenback) [reply src/components/ActivityCollector/configValidators.js:40]: I can merge this and update it later

File: src/utils/validation/createRenamedValidator.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import isObject from "../isObject";
+import { assertValid } from "./utils";
+
+export default (oldField, oldSchema, newField) =>
```
Comment (liljenback) [inline src/utils/validation/createRenamedValidator.js:15]: So this is basically the previously deprecated validator right?

**Pull request merged**